README.md

```diff
@@ -1 +1,50 @@
-# hello-world
\ No newline at end of file
+
+# Project Name: "Hello Microverse"
+
+This project is mainly about setting up diffrent development tools and working on gitHub workflows. 
+
+
+## Built With
+
+- Major languages: HTML,CSS
+- Technologies used: linters, Lighthouse, Webhint, Stylelint
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- web browser 
+### Usage
+- clone this repository into your local machine
+- unzip it
+- open the 'index.html' file with your browser
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you add some hints about getting your project locally to make your documentation more professional. I suggest you use this [template](https://github.com/tresorsawasawa/MyPortfolio/blob/develop/README.md) to do so and your documentation will look more professional.